### ISSUE TITLE
Warning messages from CallComposite component

### ISSUE THREAD

**Issue by jerry-chuang-qhr** [Azure/communication-ui-library#2468]
**Describe the bug; what happened?**

When you join a call with call composite with dev tools open, there are some warnings in the console:

```
Some icons were re-registered. Applications should only call registerIcons for any given icon once. Redefining what an icon is may have unintended consequences. Duplicates include: 
GlobalNavButton, ChevronDown, ChevronUp, Edit, Add, Cancel, More, Settings, Mail, Filter (+ 25204 more)
```

```
vendors~main.e1ece266.iframe.bundle.js:6 MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 stateChanged listeners added. Use emitter.setMaxListeners() to increase limit
    at _addListener (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:1931394)
    at EventEmitter.addListener (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:1934965)
    at CallContext.onStateChange (https://azure.github.io/communication-ui-library/main.081f3dbe.iframe.bundle.js:1:1015632)
    at AzureCommunicationCallAdapter.onStateChange (https://azure.github.io/communication-ui-library/main.081f3dbe.iframe.bundle.js:1:1040313)
    at https://azure.github.io/communication-ui-library/main.081f3dbe.iframe.bundle.js:1:1070967
    at Ii (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:2388230)
    at Vj (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:2406963)
    at exports.unstable_runWithPriority (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:2480744)
    at cg (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:2350186)
    at Dj (https://azure.github.io/communication-ui-library/vendors~main.e1ece266.iframe.bundle.js:6:2406725)
```

**What are the steps to reproduce the issue?**

This can be reproduced in the example page:
https://azure.github.io/communication-ui-library/?path=/story/composites-call-joinexistingcall--join-existing-call

1. Filling in azure user id, valid token, and call locator
2. Join the call
-  The warnings are available in devtool console

**What behavior did you expect?**

There should be no warnings in the console log

**If applicable, provide screenshots:**

![Screen Shot 2022-10-28 at 12 52 53 PM](https://user-images.githubusercontent.com/61984145/198721328-365e30ad-7897-4b9d-ba1c-d38eecbfac9b.png)

**In what environment did you see the issue?**

n/a since this can be reproduced in the official example 
https://azure.github.io/communication-ui-library/?path=/story/composites-call-joinexistingcall--join-existing-call

**Is there any additional information?**

Hello, we were looking to reduce the amount of warnings in our application's console and realized they're coming from CallComposite and can be reproduced in the example page. Would really appreciate it if anyone can take a look.

**Comment by carocao-msft** [comment]
Hi, Thanks for raising the issue, we are looking into it :)

**Comment by carocao-msft** [comment]
Hi we have merge the change required for the Max Listener exceed warning, still working on the 'Icon being re-registered' one, please stay tuned on our next release! :)

**Comment by carocao-msft** [comment]
Hi We have merged the change required for the icon re-registered warning https://github.com/Azure/communication-ui-library/pull/2506. The changes will be reflected in our next release, please stay tuned! :)

**Comment by mgamis-msft** [comment]
Hi @jerry-chuang-qhr, the fix for this issue has been released in version [1.4.1-beta.1](https://www.npmjs.com/package/@azure/communication-react/v/1.4.1-beta.1). Please try it out. Let us know if the issue still exist and we will re-open this issue.
